### DISCUSSION TITLE
Lets try and accept

### DISCUSSION THREAD

**Discussion by zerox-alpha** [Schweinepriester/github-profile-achievements#939] [General]
Lets try and accept

**Comment by im-shubh** [comment]
Okay accept this

**Comment by im-shubh** [comment]
There are total 11 but only 7 badges you can earn right now on Github :
1. Starstruck
2. Quickdraw
3. Pair Extraordinaire
4. Pull Shark
5. Galaxy Brain
6. YOLO
7. Public Sponsor

How to earn them?
1. Starstruck :
A Github account get this badge by having a repository with total 16 stars (you need to earn 16 starts on a single repository to earn this badge)

2. Quickdraw :
You can earn this badge by closing an Issue or pull request within 5 mins of opening. (It's east to get, create an issue and close that issue immediately)

3. Pair Extraordinaire :
You need to use Github desktop for this badge, during creating a commit on a pull request add a co-author and hit commit you will get this badge

4. Pull Shark :
You will get this badge once your 2 pull requests gets merged

5. Galaxy Brain :
2 accepted answers in Github discussions

6. YOLO:
You can get this badge by adding a reviewer to your pull request but you merge that request without review, (you need to add a contributor to that repository, once he accepts the invitation you can add them as a reviewer and after that merge the request before they review)

7: Public Sponsor:
You can earn this badge by sponsoring an open source or repository

Personal Opinions
Personally i think all badges are cool I've got 3 of them (Pull Shark ,Quickdraw, YOLO), and they really do enhance your profile's look. Plus it shows that you are keeping your profile updated so if you want them go for it..

For more info I'm adding this link, if you want to learn more about the badges
[https://github.com/drknzz/GitHub-Achievements](https://github.com/Schweinepriester/github-profile-achievements/discussions/url)

Mark this as answer if you feel, my comment was useful. Have a good day 😀

**Comment by LaFilleDuCode** [comment]
ok
